examples/integrations/sam/meta_sam_labelbox_video.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        data_rows=data_row_ids,
</pre><p>instead of fetching data row ids by global key and setting<code>data_rows</code>,you can set <code>global_keys=[global_key]</code> directly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>